src/main/assembly/dist/bin/easy-bag-store

```diff
@@ -14,6 +14,7 @@ else
     LOGCONFIG=$APPHOME/cfg/logback.xml
 fi
 
+LC_ALL=en_US.UTF-8 \
```

Comment:
Zodat `easy-bag-store` command line interface ook Chinese filenamen goed wegschrijft.

Reply:
Maybe, not sure of all the ramifications yet.